modules/dailymotionBidAdapter.js

```diff
@@ -0,0 +1,75 @@
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'dailymotion',
+  gvlid: 573,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   * The only mandatory parameters for a bid to be valid are the api_key and position configuration entries.
+   * Other parameters are optional.
+   *
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: () => {
+    const dmConfig = config.getConfig('dailymotion');
```

Comment:
as noted below; don't call getConfig

Reply:
Hello @ChrisHuie, we are not using `getConfig` for ad unit params anymore as recommended here https://github.com/prebid/Prebid.js/pull/10970#discussion_r1501351540.

Also, we have updated our documentation to specify that the config should be set using `pbjs.setBidderConfig`.

Is there a better way to retrieve the parameters set by the call to `pbjs.setBidderConfig`?

I saw that this type of call is already used in other adapters like adtarget or appier.

Reply:
Hello @patmmccann, @dgirardi, @ChrisHuie, following's @sebmil-daily last week, could we have some additional informations on why `getConfig` is a blocker for this to move forward please?

Reply:
In this case it's for consistency, publishers are used to provide this sort of data through [bid parameters](https://docs.prebid.org/dev-docs/bidder-adaptor.html#bidder-adaptor-Designing-your-Bid-Params). An example of an adapter that takes an API key: [revcontent](https://github.com/prebid/Prebid.js/blob/c27f64b1e059c83706435db0b1eeffc88e9b3559/modules/revcontentBidAdapter.js#L33-L35)

Reply:
Thanks @dgirardi, we'll change it now

Reply:
@dgirardi we are now using bid params. Let us know if this works for you